asistente_ladm_col/gui/qgis_model_baker/dlg_import_schema.py

```diff
@@ -382,7 +382,7 @@ def on_stderr(self, text):
         self.advance_progress_bar_by_text(text)
 
     def on_process_started(self, command):
-        # color_log_text(command, self.txtStdout)
+        color_log_text(command, self.txtStdout)
         self.txtStdout.append(command)
```

Comment:
`color_log_text()` has its own call to `line_edit.append()`. Perhaps that's why we were not calling it from `on_process_started()`, but don't worry. I'll take care of this.